app/_locales/pt_BR/messages.json

```
@@ -3,105 +3,180 @@
+  },
+  "acceptTermsOfUse": {
+    "message": "Eu li e concordo com $1",
+    "description": "$1 is the `terms` message"
+  },
+  "accessAndSpendNotice": {
+    "message": "$1 pode acessar e gastar até esse valor máximo",
+    "description": "$1 is the url of the site requesting ability to spend"
   },
   "accessingYourCamera": {
-    "message": "Acessando a sua câmera..."
+    "message": "Acessando sua câmera..."
   },
   "account": {
     "message": "Conta"
   },
   "accountDetails": {
-    "message": "Detalhes da Conta"
+    "message": "Detalhes da conta"
   },
   "accountName": {
-    "message": "Nome da Conta"
+    "message": "Nome da conta"
   },
   "accountOptions": {
-    "message": "Opções da Conta"
+    "message": "Opções da conta"
   },
   "accountSelectionRequired": {
     "message": "Você precisa selecionar uma conta!"
   },
+  "active": {
+    "message": "Ativo"
+  },
+  "activity": {
+    "message": "Atividade"
+  },
   "activityLog": {
-    "message": "registro de atividade"
+    "message": "log de atividades"
+  },
+  "addAccount": {
+    "message": "Adicionar uma conta"
   },
   "addAcquiredTokens": {
-    "message": "Adicione os tokens que você adquiriu usando o MetaMask"
+    "message": "Adicionar os tokens que você adquiriu usando o MetaMask"
   },
   "addAlias": {
-    "message": "Adicionar pseudônimo"
+    "message": "Adicionar alias"
   },
   "addNetwork": {
-    "message": "Adicionar Rede"
+    "message": "Adicionar rede"
   },
   "addRecipient": {
-    "message": "Adicionar Destinatário"
+    "message": "Adicionar destinatário"
   },
   "addSuggestedTokens": {
-    "message": "Adicionar Tokens Sugeridos"
+    "message": "Adicionar tokens sugeridos"
   },
   "addToAddressBook": {
-    "message": "Adicionar à lista de endereços"
+    "message": "Adicionar ao catálogo de endereços"
   },
   "addToAddressBookModalPlaceholder": {
-    "message": "Exemplo: John D."
+    "message": "p.ex.: John D."
   },
   "addToken": {
     "message": "Adicionar token"
   },
   "addTokens": {
-    "message": "Adicionar Tokens"
+    "message": "Adicionar tokens"
   },
   "advanced": {
-    "message": "Avançado"
+    "message": "Avançadas"
   },
   "advancedOptions": {
     "message": "Opções avançadas"
   },
   "advancedSettingsDescription": {
-    "message": "Acesse recursos do desenvolvedor, baixe registros de estado, redefina a conta, configure testnets e personalize RPC"
+    "message": "Acessar recursos de desenvolvedor, baixar logs, redefinir conta, configurar testnets e RPC personalizadas"
+  },
+  "affirmAgree": {
+    "message": "Concordo"
+  },
+  "aggregatorFeeCost": {
+    "message": "Taxa de rede do agregador"
+  },
+  "alertDisableTooltip": {
+    "message": "Isso pode ser alterado em \"Configurações > Alertas\""
+  },
+  "alertSettingsUnconnectedAccount": {
+    "message": "Navegar por um website com uma conta não conectada selecionada"
```

I believe the verb tense here is wrong. In the original English message, we have `Browsing a website`, using present continuous (the action is happening atm).

In this case, I believe the right tense for 'Navegar' would be 'Navegando'. So we should have:
```
Navegando por um website com uma conta não conectada selecionada
```

---

Thanks! I'll apply that suggestion:
```suggestion
    "message": "Navegando por um website com uma conta não conectada selecionada"
```